mojo/core/node_controller.h

```diff
@@ -277,6 +286,7 @@ class MOJO_SYSTEM_IMPL_EXPORT NodeController : public ports::NodeDelegate,
   Core* const core_;
   const ports::NodeName name_;
   const std::unique_ptr<ports::Node> node_;
+  base::RepeatingCallback<void()> tcp_success_callback_;
```

Comment:
I think it is not appropriate for NodeController to have this callback function as a member variable. And why is it a RepeatingCallback type? Wouldn't it be better OnceCallback?

Reply:
OnceCallback::Run() may only be invoked on a non-const rvalue, i.e. std::move(callback).Run().
If I declare base::OnceCallback<void()> tcp_success_callback_; at node_controller
and use it in NodeController::OnAcceptInvitation,
then it may be possible. Let me try it.